src/features/schedule-edit/model/schedule.schema.ts

```diff
@@ -40,15 +40,19 @@ export const AppointmentScheduleStep1Schema = BaseScheduleSchema
 // 약속 신청 Step 2 (추가 정보)
 export const AppointmentScheduleStep2Schema = z.object({
   requesterName: z.string().min(1, '이름은 필수입니다'),
-  requesterEmail: z.string().email('올바른 이메일을 입력하세요'),
+  requesterEmail: z.string().refine((val) => val === '' || /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(val), {
+    message: '올바른 이메일을 입력하세요',
+  }),
```

Comment:
이메일 형식을 원하시는거면 아래처럼도 가능합니다
```suggestion
z.string().email({message: '올바른 이메일을 입력하세요'})
```

Reply:
회원일때는 빈배열을 보내야해서 저렇게 적었습니다.